integration_tests/setup_test.go

```diff
@@ -346,21 +349,30 @@ func (s *IntegrationTestSuite) initGenesis() {
 	s.Require().NoError(err)
 	appGenState[genutiltypes.ModuleName] = bz
 
-	var allocationGenState types.GenesisState
-	s.Require().NoError(cdc.UnmarshalJSON(appGenState[types.ModuleName], &allocationGenState))
+	var allocationGenState allocationtypes.GenesisState
+	s.Require().NoError(cdc.UnmarshalJSON(appGenState[allocationtypes.ModuleName], &allocationGenState))
 
-	allocationGenState.Cellars = []*types.Cellar{
+	allocationGenState.Cellars = []*allocationtypes.Cellar{
 		{
 			Id: hardhatCellar.String(),
-			TickRanges: []*types.TickRange{
+			TickRanges: []*allocationtypes.TickRange{
 				{Upper: 600, Lower: 300, Weight: 900},
 			},
 		},
 	}
 	allocationGenState.Params.VotePeriod = 30
 	bz, err = cdc.MarshalJSON(&allocationGenState)
 	s.Require().NoError(err)
-	appGenState[types.ModuleName] = bz
+	appGenState[allocationtypes.ModuleName] = bz
+
+	var corkGenState corktypes.GenesisState
+	s.Require().NoError(cdc.UnmarshalJSON(appGenState[corktypes.ModuleName], &corkGenState))
+	corkGenState.CellarIds = &corktypes.CellarIDSet{
+		Ids: []string{hardhatCellar.Hex()},
```

Comment:
The cellar ID here is using `hardhatCellar.Hex()` and allocation is using `hardhatCellar.String()`. Is there a meaningful difference? Are we using different types on the chain side to represent these values?

Reply:
It looks like it's the same thing

https://github.com/ethereum/go-ethereum/blob/7231b3efb8095d3dd18d7164c3fa84d7705759d3/common/types.go#L234-L242